mod.ts

```diff
@@ -113,14 +113,15 @@ export {
   OverwriteType,
   OverrideType
 } from './src/types/channel.ts'
-export type {
-  OverwriteAsOptions,
-  OverwritePayload,
-  EmbedPayload
-} from './src/types/channel.ts'
+export type { EmbedPayload } from './src/types/channel.ts'
```

Comment:
why is it separated from other channel types

Reply:
`export type`
![image](https://user-images.githubusercontent.com/43033058/125164833-e73e1a80-e1b1-11eb-96b9-1fdad4c7f5e7.png)

Reply:
check the bottom

Reply:
fixed